FakeNFT/Scenes /Profile/EditProfileViewController.swift

```diff
@@ -0,0 +1,355 @@
+            mainStackView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            mainStackView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16),
+            
+            nameTextField.heightAnchor.constraint(equalToConstant: LayoutConstants.rowHeight),
+            descriptionTextView.heightAnchor.constraint(equalToConstant: LayoutConstants.descriptionHeight),
+            websiteTextField.heightAnchor.constraint(equalToConstant: LayoutConstants.rowHeight),
+            
+            saveButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -16),
+            saveButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            saveButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16),
+            saveButton.heightAnchor.constraint(equalToConstant: LayoutConstants.rowHeight),
+            activityIndicator.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            activityIndicator.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+            
+        ])
+    }
+    
+    private func setupNavigationController() {
+        let backButton = UIBarButtonItem(
+            image: UIImage(systemName: "chevron.left"),
+            style: .plain,
+            target: self,
+            action: #selector(didTapBackButton)
+        )
+        backButton.tintColor = .black
+        navigationItem.leftBarButtonItem = backButton
+    }
+    
+    private func fillData() {
+        nameTextField.text = initialName
+        descriptionTextView.text = initialDescription
+        websiteTextField.text = initialWebsite
+        
+        if let url = URL(string: initialAvatarURL) {
+            avatarImageView.kf.setImage(with: url)
+        }
+    }
+    
+    private func showChangePhotoAlert() {
+        let alert = UIAlertController(title: NSLocalizedString("Edit.changePhotoTitle", comment: ""), message: nil, preferredStyle: .actionSheet)
+        
+        let changePhotoAction = UIAlertAction(title: NSLocalizedString("Edit.changePhotoAction", comment: ""), style: .default) { [weak self] _ in
+            self?.showURLInputDialog()
+        }
+        let deletePhotoAction = UIAlertAction(title: NSLocalizedString("Edit.deletePhotoAction", comment: ""), style: .destructive) { [weak self] _ in
+            self?.currentAvatarURL = ""
+            self?.avatarImageView.image = UIImage(systemName: "person.crop.circle.fill")
+            self?.avatarImageView.tintColor = .systemGray
+        }
+        
+        
+        
+        let cancelAction = UIAlertAction(title: NSLocalizedString("Common.cancel", comment: ""), style: .cancel)
+        
+        alert.addAction(changePhotoAction)
+        alert.addAction(deletePhotoAction)
+        alert.addAction(cancelAction)
+        
+        present(alert, animated: true)
+    }
+    
+    private func showURLInputDialog() {
```

Comment:
```suggestion
    private func showURLInputDialog() {
```
showChangePhotoAlert и showURLInputDialog содержат повторяющийся код (UIAlertController, cancelAction). Можно вынести в helper: func presentAlert(title: String, message: String? = nil, actions: [UIAlertAction])